airflow/providers/amazon/aws/operators/athena.py

```diff
@@ -33,6 +33,9 @@ class AthenaOperator(BaseOperator):
     """
     An operator that submits a presto query to athena.
 
+    Note: if the task is killed while it runs, it'll cancel the athena query that was launched,
```

Comment:
change to `.. note:`

Similar to https://github.com/apache/airflow/blob/9fa75aaf7a391ebf0e6b6949445c060f6de2ceb9/airflow/operators/bash.py#L80

(for some reason it doesn't allow me to suggest changes)